en/firefox/new/desktop.ftl

```diff
@@ -0,0 +1,176 @@
+firefox-desktop-download-backed-by-the = Backed by the <strong>non-profit</strong> that puts people first
+firefox-desktop-download-challenging-the-status = Challenging the status quo since 1998
+firefox-desktop-download-your-privacy-comes = Your privacy comes first
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
+firefox-desktop-download-keep-all-your = Keep all your favorite browser features — and <strong>discover</strong> new ones.
+# Gmail and Docs are product names and should not be translated
+firefox-desktop-download-all-your-favourite = All your favorite { -brand-name-google } tools (like Gmail and Docs) work seamlessly in the { -brand-name-firefox-browser }.
+firefox-desktop-download-facebook-container = { -brand-name-facebook-container }
+firefox-desktop-download-download-this-browser = Download this browser extension</a> to stop { -brand-name-facebook } (and { -brand-name-instagram }) from tracking you around the web.
+firefox-desktop-download-sync-your-devices = Sync your devices
+firefox-desktop-download-screenshots = Screenshots
+# Enhanced Tracking Protection is a feature names and so is capitalized in English
+firefox-desktop-download-enhanced-tracking-protection = Enhanced Tracking Protection (ETP)
+firefox-desktop-download-from-watching-a = From watching a web tutorial to keeping an eye on your favorite team, your video follows you while you multitask.
+firefox-desktop-download-firefox-browser = { -brand-name-firefox-browser }
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
+firefox-desktop-download-get-firefox-android = Get { -brand-name-firefox-browser } for <strong>{ -brand-name-android }</strong>
+firefox-desktop-download-get-firefox-ios = Get { -brand-name-firefox-browser } for <strong>{ -brand-name-ios }</strong>
+firefox-desktop-download-download-the-mobile = Download the { -brand-name-firefox } mobile browser for automatic protection on all your devices.
+firefox-desktop-download-learn-about-the = Learn about the { -brand-name-firefox } desktop browser below
+
+# Variables:
+#   $attrs (attrs) - link to https://addons.mozilla.org/firefox/extensions/
+firefox-desktop-download-from-security-to = From security to news to gaming, there’s an <a { $attrs }>extension for everyone</a>. Add as many as you want until your browser is just right.
```

Comment:
Please add a comment on what it means `your browser is just right`. What is the alternative?